packages/jentic-openapi-validator-redocly/tests/conftest.py

```
@@ -86,7 +86,7 @@ def pytest_runtest_setup(item):
     if item.get_closest_marker("requires_redocly_cli"):
         try:
             result = subprocess.run(
-                ["npx", "--yes", "@redocly/cli@V2.15.1", "--version"],
+                ["npx", "--yes", "@redocly/cli@2.15.1", "--version"],
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)